README.md

```diff
@@ -28,22 +28,40 @@ server:
 keycloak:
   baseURL: "http://localhost:8080"
   realm: "master"         
-  clientID: "admin-cli"
+  # For admin operations
+  adminClientID: "admin-cli"
+  adminUsername: "admin"
+  adminPassword: "admin"
+  # For user operations
+  clientID: "identity-service"
+  clientSecret: "your-client-secret" # See step 5
   adminUsername: "admin"
   adminPassword: "admin"
 ```
 
 4. Start keycloak server
 ```bash
-docker-compose up
+docker compose up
 ```
 
-5. Run the application
+5. Configure Keycloak
+   Follow the [Keycloak Setup Guide](docs/KEYCLOAK_SETUP_GUIDE.md) to:
+   - Create a custom realm
+   - Configure the required client
+   - Set up proper authentication flows
```

Comment:
heads up, found a way to automate this, has a bug rn, but will automate this step entirely.

Reply:
Woohoo, then this can be deleted later. Watcha using?

Reply:
docker compose will call a keycloak-init.sh script which will do these steps using curl commands, saves headache and will update a .env file with the values (can be changed to config.yaml if we want :) )